manifests/kustomize/base/pipeline/ml-pipeline-scheduledworkflow-deployment.yaml

```diff
@@ -22,4 +22,9 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          #- name: CRON_SCHEDULE_TIMEZONE
```

Comment:
I have some issues when I add the configurations for the env variable but don't know where that come from ....